angularjs-google-maps/ng-map.d.ts

```diff
@@ -0,0 +1,148 @@
+﻿// Type definitions for angularjs-google-maps v1.17.3
+// Project: https://github.com/allenhwkim/angularjs-google-maps
+// Definitions by: Niko Kovačič <https://github.com/nkovacic>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
```

Comment:
Can you fix this line to use the new URL `https://github.com/DefinitelyTyped/DefinitelyTyped`